deployments/kubernetes/chart/gitwebhookproxy/templates/ingress.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    fabric8.io/generated-by: {{ .Values.gitWebhookProxy.ingress.annotations.generatedBy }}
+    ingress.kubernetes.io/force-ssl-redirect: "{{ .Values.gitWebhookProxy.ingress.annotations.forceSslRedirect }}"
```

Comment:
make these annotations optional, or maybe take all annotations key values from value. @waseem-h what do you suggest?

Reply:
Updated.